docs/source/tutorials/faq.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ffe0d62e",
+   "metadata": {},
+   "source": [
+    "# FAQ\n",
+    "\n",
+    "Answers to frequently asked questions about the [cleanlab](https://github.com/cleanlab/cleanlab) open source package"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eca36874",
+   "metadata": {},
+   "source": [
+    "### How do I format labels for Cleanlab?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5d0fbb3",
+   "metadata": {},
+   "source": [
+    "Cleanlab only works with integer-encoded labels in the range `{0,1, ... K-1}` where `K = number_of_classes`. Final labels array should only contain integer values in the range  `{0, K-1}` and be of shape `(N,)` where `N = total_number_of_data_points`.\n",
+    "\n",
+    "**Text labels** should to be integer-encoded. An example of text to integer-encoding can be found in step 2 of [Text Classification with TensorFlow, Keras, and Cleanlab Tutorial](https://docs.cleanlab.ai/v2.0.0/tutorials/text.html#2.-Load-and-preprocess-the-IMDb-text-dataset). \n",
+    "\n",
+    "**One-hot encoded labels** should be integer-encoded by finding the argmax along the one-hot encoded axis. An example of what this might look like is shown below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "239d5ee7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np \n",
+    "\n",
+    "arr  = np.array([[0,1,0,0],[1,0,0,0],[0,0,0,1]]) # [3,4] array wih 4 labels\n",
```

Comment:
Comment should clarify: 

This example has 3 labels (one per datapoint) where each label can be one of 4 possible classes.

Reply:
I'd be wary of using 3 examples though and 4 classes, since cleanlab is not intended to be used with more classes than data, and there will coincidentally happen to be two instances of the number 3 (one for # examples, one for max-class-integer).

I'd instead consider changing this example to something like 4 examples, 3 classes.